frontend/src/components/translation/TranslationProgressBar.tsx

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import { ProgressBar } from "react-bootstrap";
```

Comment:
we haven't made any formal component library decisions so I'm happy with keeping the bootstrap. I am curious, do you have a particular reason for bootstrap or are you comfortable with it?

Reply:
oops i can definitely swap it out later once we decide on a component library! not too much thought behind it, but thought it would be easier to quickly spin up a functional progress bar
i thought i had saw bootstrap used elsewhere in the repo but i think i just imagined it now 😅
i have used it a few times before but definitely can switch it to something else!!